doc/api/esm.md

```diff
@@ -909,16 +874,17 @@ prints the current version of CoffeeScript per the module at the URL in
 #### Transpiler loader
 
 Sources that are in formats Node.js doesn’t understand can be converted into
-JavaScript using the [`transformSource` hook][]. Before that hook gets called,
-however, other hooks need to tell Node.js not to throw an error on unknown file
-types; and to tell Node.js how to load this new file type.
+JavaScript using the [`load` hook][load hook]. Before that hook gets called,
+however, a [`resolve` hook][resolve hook] hook needs to tell Node.js not to
+throw an error on unknown file types.
 
 This is less performant than transpiling source files before running
 Node.js; a transpiler loader should only be used for development and testing
 purposes.
 
 ```js
 // coffeescript-loader.mjs
+import { readFile } from 'fs/promises';
```

Comment:
So I think now we just need to update this from the example in the other repo?

Reply:
Yes? Weren't we going to move the other repo into nodejs? If we do, should we copy+paste those example(s) or link to them here?